charts/telescope-backend/files/db_tables.sql

```diff
@@ -21,7 +21,7 @@ SET default_tablespace = '';
 SET default_table_access_method = heap;
 
 --
--- Name: capability; Type: TABLE; Schema: public; Owner: telescope
```

Comment:
Should the owner be changed from `telescope` to `postgres`?

Reply:
In the db calls, they all use the 'telescope' user not the default postgres user.